contracts/exchange-libs/contracts/src/LibOrder.sol

```diff
@@ -133,6 +133,11 @@ library LibOrder {
         // ));
 
         assembly {
+            // Assert order offset (this is an internal error that should never be triggered)
```

Comment:
Why is this just 32? Couldn't it be at least 64? It seems like it should be:  `lt(order, 0x80)`?